app/modifiers/focus-when.js

```diff
@@ -0,0 +1,7 @@
+import { modifier } from 'ember-modifier';
+
+export default modifier(function focusWhen(element, [isFocused]) {
```

Comment:
- should we change the name to something else from `foucsWhen` as it's not very descriptive, what about `setFocus` as this sets the focus on a element.